src/pybind/mgr/cephadm/services/nvmeof.py

```diff
@@ -171,7 +171,7 @@ def get_set_cmd_dicts(out: str) -> List[dict]:
                         'prefix': 'dashboard nvmeof-gateway-add',
                         'inbuf': service_url,
                         'name': service_name,
-                        'group': spec.group,
+                        'group': spec.group if spec.group else '',
```

Comment:
@adk3798 is there any other name (like `default`) that is set to the group if there is no group?

Reply:
I don't think so. Iirc I tried to change it so that it was `default` if one wasn't provided in the past and was asked to change it back to an empty string (cc @caroav )